src/namespaces.json

```diff
@@ -198,5 +198,17 @@
     "min": 0,
     "invalid": [],
     "visible": false
+  },
+  "yflink": {
+    "key": "yflink",
+    "symbol": "YFL",
+    "name": "YFLink",
+    "defaultView": "core",
+    "address": "0x28cb7e841ee97947a86b06fa4090c8451f64c0be",
```

Comment:
The addrsss must be checksumed, same for "token" param

Reply:
Done. Thanks!